StripePaymentSheet/StripePaymentSheet/Source/Helpers/PaymentSheetLinkAccount.swift

```diff
@@ -238,19 +236,19 @@ class PaymentSheetLinkAccount: PaymentSheetLinkAccountInfoProtocol {
     func createLinkAccountSession(
         completion: @escaping (Result<LinkAccountSession, Error>) -> Void
     ) {
-        guard let session = currentSession else {
-            stpAssertionFailure()
-            completion(
-                .failure(
-                    PaymentSheetError.unknown(
-                        debugDescription: "Linking account session without valid consumer session"
+        retryingOnAuthError(completion: completion) { [publishableKey] completionWrapper in
```

Comment:
Is there a concern that the publishableKey will be changing after making this call? I think this might be introducing more complexity than needed?

Reply:
The reformatting of this file is making the diff confusing, but I'm only moving the `guard let session = currentSession` into the retryingOnAuthError block: The session could change after making this call, as `retryingOnAuthError` could create a new session and change the `currentSession`.

Reply:
I'll remove the explicit captures and just use `self` in these functions, maybe that'll make it less confusing.